### ISSUE TITLE
Semantics Feedback

### ISSUE THREAD

**Issue by Enspired** [afarnha42/GDMA-2567-Exercise-1#1]
- [x] - Div line 18 can be the header, and the header line 19 can be a div 
- [x] - Div line 10 does not need a wrapper class because it’s extending to both edges of the browser 
- [x] - Div line 40 needs a wrapper class 
- [x] - Div line 49 should be the section, lines 50 & 55 should be divs 
- [x] - Line 51 can be a headline, include header tags 
- [x] - Line 55 will need a col- class for large display. You will also need to work with positioning here 
- [x] - Missing col- class around header line 70, row class can be on the same element 
- [x] - Line 74 is not a semantic element, the section grouping all content together is already at line 69 
- [x] - Lines 75, 84, 93 need to be semantic elements. Include headline and header tags 
- [x] - Divs line 76, 85, 94 should be image tags (will require less code) 
- [x] - Line 77, 86, 95 should be two text tags, a paragraph and headline 
- [x] - Content lines 78-82 cannot be a UL. You can’t structure one li to go to the left and the last two to go to the right. These will need to be paragraphs that are floated with divs. Utilize your col- classes and row. Update for all identical ULs 
- [x] - Line 107, this background is outside the wrapper so will go to 100% by default. A col- class is not needed here 
- [x] - Line 108, this is not a section (missing headline). Can be a div 
- [x] - Blockquote can wrap around the image, col- class can be applied to the blockquote 
- [x] - Missing footer tags within the blockquote 
- [x] - Encode emdash 
- [x] - Line 117 is not a div 
- [x] - Missing col- class around header line 118, row class can be on the same element 
- [x] - Line 122 is not a section 
- [x] - Lines 123, 130, 137, 175, 184, 193 need to be semantic elements 
- [x] - Line 154, col- class is not needed for the background color. (I believe this is because of a question Hanna asked. She and I must have had some miscommunication. I didn’t realize she was asking about background colors/images that span to 100% of the browser window) 
- [x] - Missing headline and header within div line 156 
- [x] - Line 161 should be inside the form to describe the input 
- [x] - Div line 160 can be replaced with the form element 
- [x] - Headlines 175-199 should not be h4’s - update all other headlines accordingly 
- [x] - Missing time tags 
- [x] - Divs line 176, 185, 194 will be easier to place as images in the HTML  
- [x] - Without a headline, line 207 is not a section 
- [x] - Link email, phone, and address respectively 
- [x] - Missing copyright info 
- [ ] 

Be sure to link up the navigation before starting any CSS

**Comment by Enspired** [comment]
- [x] - Don’t forget to include font awesome icons for line 34, 37 
- [x] - Line 41, if the image goes in through the HTML, the ID can be removed 
- [x] - Format HTML properly, line 51. H2 should be indented inside the header 
- [x] - Sections line 75, 95, 115 are the incorrect semantic element. They are independent of the popular courses section. They could be deleted from the parent section and no value would be lost. 
- [x] - Divs line 77, 97, 117 should be semantic elements 
- [x] - Missing headlines within the popular courses sub-sections 
- [x] - The strike through price and actual price lines 86-91 always sit next to each other. These two paragraphs can be in one div together, simply manipulate the display or create a list. Update for all other popular courses 
- [x] - Line 141 can be inside the blockquote since it is related to the content
- [x] - Missing text tag around the content at line 144 
- [x] - Incorrect col- class size on the blockquote and lines 150, it does not span all 12 columns at all sizes. The col- class size should be used to control the overall width of the text. 
- [x] - Line 149 should be a semantic element 
- [x] - Lines 206, 215, 224 should be semantic elements 
- [x] - Missing headers and headlines for each blog post 
- [x] - Datetime attribute value is not written correctly, this will throw validation errors
- [x] - Extra divs lines 208, 217, 226 
- [x] - Don’t forget links and text for the social icons 
- [x] - Line 278, all content needs to be contained within a wrapper 

Before beginning any CSS, be sure the navigation and all buttons are linked up accordingly, review all responsive structure / layout, address any validation errors.

**Comment by Enspired** [comment]
- [ ] - Classes should be used more than once, otherwise they’ll need to be IDs 
- [ ] - Divs line 77, 101, 124 should be semantic elements. 
- [ ] - Sections line 79, 103, 126 can be removed 
- [ ] - Paragraphs line 80, 102, 104 can be included in the header tag since these are describing the type of class 
- [ ] - Missing col- classes line 162 
- [ ] - Label / input utilize an ID value that provides context to the form input 
- [ ] - Incorrect semantic element line 218, 229, 240. These blog posts can be removed or swapped out at any time without impacting the communication of the post. 
- [ ] - Social icons are incomplete. They need to be clickable and include text for screen readers. 
- [ ] - Div line 296 is missing the wrapper class 
- [ ] - Font-sizes applied to the HTML element should be set in REMs
